### DISCUSSION TITLE
Fragmented lists and pdf conversion

### DISCUSSION THREAD

**Discussion by timmens** [marp-team/marp#311] [Q&A] [answered]
When converting to pdf, slides with fragmented lists become one slide. Is there any way to render it so they become multiple pages?

**Comment by yhatt** [comment, marked as answer]
Marp CLI has no abillity to get it. PDF output cannot reproduce any animations (Marp treats the fragmented list as animation, just like bullet animations in PowerPoint). I've proposed the solution in marp-team/marp-cli#141, but we are not working on it due to the problem of slide reproducibility when used GIF and `<video>`.

[decktape](https://github.com/astefanutti/decktape) PDF exporter for general HTML slides may help to render fragmented list correctly.

```bash
marp your-markdown.md

# Load pause has required to wait hiding on-screen controller
decktape generic -s 1280x720 --load-pause 3000 ./your-markdown.html ./your-markdown.pdf
```

**Reply by timmens** [reply]
Ah I see. decktape actually solved the problem. Thanks a lot!
